driver-core/src/test/functional/com/mongodb/ClusterFixture.java

```diff
@@ -143,6 +144,47 @@ public static boolean serverVersionGreaterThan(final String versionString) {
         return getServerVersion().compareTo(new ServerVersion(getVersionList(versionString).subList(0, 3))) > 0;
     }
 
+    public static boolean canRunTests(final BsonArray runOn) {
```

Comment:
This method doesn't seem to fit with the rest of ClusterFixture.  I'd prefer it move somewhere else, even if that means adding a new Helper class

Reply:
Moved to a new file